__tests__/functions/functions.ts

```diff
@@ -0,0 +1,48 @@
+import { cleanPercentage } from '@/src/components/circular-progress';
+
+describe("cleanPercentage", () => {
+  it("should return a number if the input is a within the values of 0-100", () => {
+    // When input is a number
+    expect(cleanPercentage(50)).toBe(50);
+
+    // When input is a string
+    expect(cleanPercentage('30')).toBe(30);
+  })
+
+  it("should return 0 for negative values", () => {
+    // When given a negative numerical value
+    expect(cleanPercentage(-15)).toBe(0);
+
+    // When given a negative alphabetic value
+    expect(cleanPercentage("-15")).toBe(0);
+  })
+
+  it("should return 100 for values greater than 100", () => {
+    // When given a negative numerical value
```

Comment:
I think the comment is misleading here. How about saying something along the lines of, "When given a numeric value greater than 100"